ingest/expression_files/dense_ingestor.py

```diff
@@ -0,0 +1,187 @@
+"""Module for ingesting dense matrix files
+
+DESCRIPTION
+Module provides extract and transforms function for gene expression data for
+an dense matrix.
+
+"""
+import collections
+import csv
+import datetime
+import sys
+from typing import List  # noqa: F401
+
+from bson.objectid import ObjectId
+
+try:
+    from expression_files import GeneExpression
+    sys.path.append("../ingest")
+    from ingest_files import IngestFiles
+
+except ImportError:
+    # Used when importing as external package, e.g. imports in single_cell_portal code
+    from .expression_files import GeneExpression
+    sys.path.append("../ingest")
+    from ..ingest_files import IngestFiles
+
+
+class DenseIngestor(GeneExpression, IngestFiles):
+    ALLOWED_FILE_TYPES = ["text/csv",
+                          "text/plain", "text/tab-separated-values"]
+
+    def __init__(self, file_path, study_file_id, study_id, **kwargs):
```

Comment:
Add a method comment.  And either rename the **kwargs param **matrix_params or add a comment explaining that kwargs will be used as matrix params

Reply:
aab53bd